r_scripts/data_wrangle/oregon_casecrossover.R

```diff
@@ -0,0 +1,106 @@
+# ------------------------------------------------------------------------------
+# Title: Case crossover for diseases (Oregon State) for server
+# Author: Jingyang Liu
+# Date Created: Nov 27, 2017
+# R version: 3.3.2
+# ------------------------------------------------------------------------------
+
+#Creating case crossover dataframes
+library(dplyr) 
+library(data.table)
+library(readxl)

```

Comment:
You don't need readxl package. Delete the line calling it to avoid dependency issues.